rustls/src/crypto/ring/tls13.rs

```diff
@@ -101,18 +101,21 @@ impl Tls13AeadAlgorithm for AeadAlgorithm {
     }
 }
 
+#[used]
```

Comment:
Can you leave a `TODO` or some kind of comment here linking to the rustc bug?

It will be hard to remember why these attributes are here in the future otherwise. One day our MSRV will have the fix and we can rip these out again :crossed_fingers:

Reply:
have managed to drop this commit and workaround entirely